modules/apim_tm_master/manifests/startserver.pp

```diff
@@ -0,0 +1,22 @@
+# ----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
```

Comment:
Copyright notices should be updated to 2019 as this is a new puppet module.

Reply:
Resolved https://github.com/wso2/puppet-apim/pull/86/commits/5dd25054bc378c8b256e4571b433b51eecc8d807